languagetool-language-modules/en/src/main/java/org/languagetool/language/BritishEnglish.java

```diff
@@ -86,4 +105,12 @@ protected int getPriorityForId(String id) {
     }
     return super.getPriorityForId(id);
   }
+
+  public static @NotNull BritishEnglish getInstance() {
+    Language language = Objects.requireNonNull(Languages.getLanguageForShortCode(LANGUAGE_SHORT_CODE));
+    if (language instanceof BritishEnglish britishEnglish) {
```

Comment:
This will not work with the close source part :( See my comments in German: https://github.com/languagetool-org/languagetool/blob/482657017ccde7123292c801ef9f332a4f6c9e80/languagetool-language-modules/de/src/main/java/org/languagetool/language/GermanyGerman.java#L90

This applies for those languages and their variants:

de
en
es
pt
fr
nl

Reply:
fixed